test/lib/widgets/ntp_server_test.rb

```diff
@@ -5,7 +5,7 @@
 
 describe Y2Caasp::Widgets::NtpServer do
   subject(:widget) { Y2Caasp::Widgets::NtpServer.new }
-  let(:dashboard_role) { ::Installation::SystemRole.new(id: "dashboard_role") }
+  let(:dashboard_role) { ::Installation::SystemRole.new(id: "dashboard_role", order: "100") }
```

Comment:
Does exist a default value for `order`? (Just asking)

Reply:
no, there was discussion and intentionally without default